content/hi/partners/_index.html

```diff
@@ -0,0 +1,88 @@
+---
+title: सहभागी
+bigheader: कूबरनेट्स सहभागी
+abstract: कूबरनेट्स इकोसिस्टम का विस्तार.
+class: gridPage
+cid: सहभागी
+---
+
+<section id="users">
+    <main>
+        <h5>कुबरनेट्स सहयोगियों संग एक मज़बूत कोड बेस का निर्माण करता है जो संपूरक मंचों के स्पेक्ट्रम को सहयोग करता है।</h5>
+        <div class="col-container">
+          <div class="col-nav">
+            <center>
+              <h5>
+                <b>कुबरनेट्स प्रमाणित सेवा प्रदाता</b>
+              </h5>
+              <br>उद्यमों द्वारा कुबरनेट्स सफलतापूर्वक अपनाने में सहभागी अनुभवी सेवा प्रदाता।
+              <br><br><br>
+              <button id="kcsp" class="button" onClick="updateSrc(this.id)">KCSP सहभागियों को देखें</button>
+              <br><br><a href="https://www.cncf.io/certification/kcsp/">KCSP बनने में दिलचस्प</a>?
+            </center>
+          </div>
+          <div class="col-nav">
+            <center>
+              <h5>
+                <b>प्रमाणित कूबरनेट्स Distributions, Hosted Platforms, तथा Installers</b>
+              </h5>सॉफ्टवेयर अनुरूप चलने से ये सुनिश्चित किया जाता है कि प्रत्येक विक्रेता का संस्करण जरूरी API का समर्थन करे।
+              <br><br><br>
+              <button id="conformance" class="button" onClick="updateSrc(this.id)">अनुरूप सहभागियों को देखें</button>
+              <br><br><a href="https://www.cncf.io/certification/software-conformance/">प्रमाणित कूबरनेट्स बनने में दिलचस्प</a>?
```

Comment:
The English version of this sentence is also not making any sense `Interested in becoming Certified Kubernetes`.

I guess it should be `Interested in becoming Kubernetes Certified`

Reply:
I think this should be changed. What do you say @Rajakavitha1 ?

Reply:
https://github.com/kubernetes/website/pull/14311

Reply:
@saarques This PR https://github.com/kubernetes/website/pull/14311 is merged, please do the needful and make necessary changes to Hindi version too.